test/lib/get-distro.js

```diff
@@ -8,12 +8,13 @@ const debug = require('debug')('logdna:test:lib:getDistro');
 const getDistro = require('../../lib/get-distro');
 
 // Constants
-const testConfig = '../assets/test-getDistro.config';
+const testConfig = 'test/assets/test-getDistro.config';
 
 describe('lib:getDistro', () => {
     it('Gets the correct Distro', () => {
         return getDistro(testConfig, (error, os) => {
             debug(os);
+            console.log(os);
```

Comment:
This line is intentional?

Reply:
Oooops... forgot to remove - now removed - done!